httpserver/http_connection.cpp

```diff
@@ -798,17 +804,17 @@ void connection_t::process_snodes_by_pk(const json& params) {
         return;
     }
 
-    auto pubKey = params["pubKey"].get<std::string>();
-
-    if (pubKey.size() != 66) {
+    bool success;
```

Comment:
I see there are other instances of this, happy with either all success or valid

Reply:
We already use success somewhere else, so had to come up with a different name above. I wish I didn't have to pollute the "namespace" with these single-use variables. I'm tempted to use `boost::optional` here.